package.json

```diff
@@ -78,8 +78,8 @@
     "@types/yargs": "^12.0.1",
     "colors": "^1.3.2",
     "fs": "^0.0.1-security",
-    "gulp": "^3.9.1",
-    "js-yaml": "^3.12.0",
+    "gulp": "^4.0.0",
```

Comment:
Is this change in the root package.json required to address the issue?  If so, we should run this by the management library team as well, since it impacts all packages in the repo.

Reply:
I'll check if I don't need to update gulp

Reply:
I've reverted my changes to root - package.json